<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>   Naming</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="lineEdit_nameInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The name your object will appear ingame. Consider the naming guide.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nameExplanation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Specify additional ingame names for other languages. For languages without name the english name will always be taken. English corresponds to 'Object Ingame Name' from Main Settings.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_languageSelect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>English - UK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>English - US</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>French</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>German</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spanish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Italian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dutch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Swedish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Japanese</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Korean</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chinese - China</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chinese - Taiwan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Polish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Portuguese</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Czech</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Russian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Esperanto</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>  Linking Options</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scenery Group ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sceneryGroupID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter ID of a scenery group such that this object gets added to it automatically.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mirror Object ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mirrorID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter ID of a mirror version of the current object for future support of correct mirroring behaviour.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
